src/lib/components/ui/table.tsx

```diff
@@ -8,7 +8,7 @@ const Table = React.forwardRef<
 >(({ className, ...props }, ref) => (
   <table
     ref={ref}
-    className={cn('w-full caption-bottom', className)}
+    className={cn('w-auto min-w-max caption-bottom', className)}
```

Comment:
Why is this min-w-max? 

Seems to cause the scrollwidth to be very large and i am able to scroll horizontally a far distance. 
When I removed it, everything works. Just want to know why it was added originally.